website/docs/release_notes.md

```diff
@@ -7,6 +7,189 @@ import ReactPlayer from 'react-player'
 
 # Release notes
 
+## [v1.0.0-RC2](https://github.com/VirtusLab/scala-cli/releases/tag/v1.0.0-RC2)
+
+## What's Changed
+
+### Exclude
+
+To exclude specific source files or entire directories from a Scala CLI project using the exclude directive in your `project.scala` file or the command line parameter `--exclude`, along with a pattern:
+- absolute path: `/root/path/to/your/project/Main.scala`
+- relative path: `src/main/scala/Main.scala`
+- glob pattern: `*.sc`
+
+For example, to exclude all files in the` example/scala` directory, add the following directive to your `project.scala` file:
+```scala
+//> using exclude "example/scala"
+```
+
+Added by [@lwronski](https://github.com/lwronski) in [#2053](https://github.com/VirtusLab/scala-cli/pull/2053).
+
+### Directives with a Test Scope Equivalent
+
+Some directives now have a test scope equivalent, such as `using dep` and its test scope counterpart `using test.dep`. This allows you to declare dependencies that are only used in tests outside of test-specific sources.
+
+For example, you can declare a dependency on munit in your `project.scala` file like this:
+```
+//> using test.dep "org.scalameta::munit::0.7.29"
+```
+The dependency will only be available in test sources.
+
+Here's a list of directives with a test scope equivalent with example values:
+
+```
+ //> using test.dep "org.scalameta::munit::0.7.29"
+ //> using test.jar "path/to/jar"
+ //> using test.javaOpt "-Dfoo=bar"
+ //> using test.javacOpt "source", "1.8", "target", "1.8"
+ //> using test.javaProp "foo1=bar1"
+ //> using test.option "-Xfatal-warnings"
+ //> using test.resourceDir "testResources"
+ //> using test.toolkit "latest"
+```
+
+Added by [@Gedochao](https://github.com/Gedochao) in  [#2046](https://github.com/VirtusLab/scala-cli/pull/2046)
+
+### Changes to using-directives syntax
+
+We've made several updates to simplify the using directives syntax in this release:
+
+- removed the possibility of nesting from the API.
+- allowed omitting commas in lists of values.
+- disallowed multiline comments.
+- removed multiline strings.
+- removed `require` and `@require` syntax support.
+- allowed values without quotes.
```

Comment:
I think it'd be awesome to include an example showing that commas and quotes are now optional.
maybe something like:
````
```scala
//> using scala 3.2.2
//> using javacOpt source 1.8 target 1.8
```
````